main-app/tests/playerSelection/player-controller.js

```diff
@@ -0,0 +1,25 @@
+(function () {
+    'use strict';
+    /*jshint expr: true*/
```

Comment:
You can keep this in your code if you want, but I'd put a proper option in the config for the exact linter message - the framework is forcing this for us:
````

    unittests: {
        src: ['./main-app/tests/**/*.js'],
        options: {
            '-W030': true,
            'globals'   : {
                'describe'   : false,
                'it'         : false,
                'before'     : false,
                'beforeEach' : false,
                'after'      : false,
                'afterEach'  : false,

            }
        }
    }
````